README.md

```diff
@@ -80,6 +80,14 @@ microk8s can be restarted later with the snap command
 snap enable microk8s
 ```
 
+### Removing microk8s
+
+To remove microk8s we recommend you first disable all addons and stop all hosted pods. microk8s.reset can assist with that.
```

Comment:
"Before removing microk8s, use `microk8s.reset` to stop all running pods."

I understand why we need this right now, but it would be much nicer if we could get a pre-remove hook for snaps and then put this logic there, so the user doesn't have to know or think about it.

Reply:
We brought this to the snapcraft forum: https://forum.snapcraft.io/t/cleanup-before-services-go-down/5802